server.js

```diff
@@ -1720,28 +1721,30 @@ if (config.startServer) {
           })
         );
       },
-      function (callback) {
-        var pgConfig = {
+      async function () {
+        const pgConfig = {
           user: config.postgresqlUser,
           database: config.postgresqlDatabase,
           host: config.postgresqlHost,
           password: config.postgresqlPassword,
           max: config.postgresqlPoolSize,
           idleTimeoutMillis: config.postgresqlIdleTimeoutMillis,
         };
-        logger.verbose(
-          'Connecting to database ' + pgConfig.user + '@' + pgConfig.host + ':' + pgConfig.database
-        );
-        var idleErrorHandler = function (err) {
+        function idleErrorHandler(err) {
           logger.error('idle client error', err);
           // https://github.com/PrairieLearn/PrairieLearn/issues/2396
           process.exit(1);
-        };
-        sqldb.init(pgConfig, idleErrorHandler, function (err) {
-          if (ERR(err, callback)) return;
-          logger.verbose('Successfully connected to database');
-          callback(null);
-        });
+        }
+
+        logger.verbose(`Connecting to ${pgConfig.user}@${pgConfig.host}:${pgConfig.database}`);
+
+        await sqldb.initAsync(pgConfig, idleErrorHandler);
```

Comment:
Does this init also need to be added to the test initialization code? https://github.com/PrairieLearn/PrairieLearn/blob/073606f8c65addc40684c12fa7d74cd98cc7c769/tests/helperServer.js#L47

Reply:
Yes! Great catch.

Reply:
Done in 099d7f9.